README.md

```diff
@@ -89,7 +101,9 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+  ERD = entity relationship diagram.  
+  There are relationships between entities like one to one, one to many, many to many.
```

Comment:
You are correct.  We were hoping you would elaborate a bit.  Can you name the relationships likely to exist between a Lamp and a Genie?  How about a Genie and a Pet?